api/migrations/20210212200719-addExpectedBudgetTimeframeToProjects.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
```

Comment:
Maybe we should add some documentation to the README as well for how to generate these migrations using sequelize. Even if it is just a link to the Sequelize documentation but some guidance would be helpful